### PR TITLE
Fix blog not building

### LINKED ISSUE #6752: Example blog doesn't work
The example blog provided in `examples/` seems to be obsolete when ran with up-to-date Hugo version:

```console
marc@cobalt:~/hugo/examples/blog$ hugo
Building sites … WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomyTerm": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomyTerm": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/01/14 10:35:41 found no layout file for "HTML" for "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
Total in 18 ms
Error: Error building site: failed to render pages: render of "home" failed: "/var/folders/qb/512rrh_n51x1fb40yc3l1sj40000gn/T/tmp.n7Ecxv4E/hugo/examples/blog/layouts/index.html:10:90": execute of template failed: template: index.html:1:3: executing "index.html" at <partial "header.html" .>: error calling partial: "/var/folders/qb/512rrh_n51x1fb40yc3l1sj40000gn/T/tmp.n7Ecxv4E/hugo/examples/blog/layouts/partials/header.html:10:90": execute of template failed: template: partials/header.html:10:90: executing "partials/header.html" at <.Title>: can't evaluate field Title in type *page.OutputFormat
```

The compilation error can be fixed with this change:

```diff
--- a/examples/blog/layouts/partials/header.html
+++ b/examples/blog/layouts/partials/header.html
@@ -7,6 +7,6 @@
     <link rel="canonical" href="{{ .Permalink }}">
     {{ partial "header.includes.html" . }}
     {{ with .OutputFormats.Get "RSS" -}}
-    {{ printf "<link href=%q rel=\"alternate\" type=%q title=%q />" .Permalink .MediaType .Title | safeHTML }}
+    {{ printf "<link href=%q rel=\"alternate\" type=%q title=%q />" .Permalink .MediaType $.Site.Title | safeHTML }}
     {{- end }}
 </head>
```

However even when fixing the compilation error, the rendered blog is broken (clicking on either `Posts` or `Read More` lead to a blank page):

Screenshot 2020-01-14 at 10 38 02

### What version of Hugo are you using (`hugo version`)?

```
$ hugo version
Hugo Static Site Generator v0.62.2/extended darwin/amd64 BuildDate: unknown
```

### Does this issue reproduce with the latest release?

Yes

### PR DESCRIPTION
**Pull request by colonelpopcorn**
The example blog had not been updated for some time and I had to restructure it a little bit.

Fixes #6752

Changed tag and category directory structure

### CODE REVIEW

File: examples/blog/layouts/partials/header.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+
 <head>
     {{ partial "meta.html" . }}
 
     <title>{{ .Title }} - {{ .Site.BaseURL }}</title>
     <link rel="canonical" href="{{ .Permalink }}">
     {{ partial "header.includes.html" . }}
     {{ with .OutputFormats.Get "RSS" -}}
-    {{ printf "<link href=%q rel=\"alternate\" type=%q title=%q />" .Permalink .MediaType .Title | safeHTML }}
+    {{ printf "<link href=%q rel=\"alternate\" type=%q title=%q />" .Permalink .MediaType $.Site.Title | safeHTML }}
```
Comment (bep) [inline examples/blog/layouts/partials/header.html]: I suspect the "correcter" fix in the above would be tu use `$.Title`.

Comment (colonelpopcorn) [reply examples/blog/layouts/partials/header.html]: Okay, I will make the change. Do you want me to do a second rebase after committing or can I amend the commit and force push to my branch?

Comment (moorereason) [reply examples/blog/layouts/partials/header.html]: Fix, squash, and force push.

**Comment by falzm** [comment]
Tested OK, thank you @colonelpopcorn!

**Pull request merged**